color.py

```diff
@@ -29,7 +29,7 @@ def is_left_black():
         return False
 
 def is_left_not_white():
-    if left_color_sensor_rli() < 90:
```

Comment:
If the reflection value is 50, should it be not black, or not white?

Reply:
The value 50 in this program area is to determine that the value is not white and is most likely the other colors on the map.